homework/index.js

```diff
@@ -29,13 +29,54 @@
     return elem;
   }
 
+  function makeText(stringDateAndTime) {
+    const dt = new Date(stringDateAndTime);
+    return dt.toLocaleString();
+  }
+
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    const li = createAndAppend('li', ul, { class: 'repos' });
+    const table = createAndAppend('table', li, { class: 'table' });
```

Comment:
An HTML table is an appropriate choice. But there some repetition of code here that begs to be put in a separate, reusable function. The repetition I see is: create a `tr` with a `td` child with some label text and another `td` child with a repository property text value. So I could imagine a function, say `addRow`, as follows:

```js
function addRow(table, labelText, value) {
  const tr = ...
  ...
  return tr;
}
```

The first row requires some special handling as it should contain a `<a>` tag instead of simple text. I'll give you a hint how this can be done:

```js
const firstRow = addRow(table, 'Repository:', '');
createAndAppend('a', firstRow.lastChild, ...));
```